cdap-api/src/main/java/io/cdap/cdap/api/macro/MacroEvaluator.java

```diff
@@ -74,11 +75,12 @@
   }
 
   /**
-   * Get the list of macro function names that need to be evaluated as map.
+   * Get the type of the macro function evaluation result
```

Comment:
Should mention that String type means evaluate() is called, while Map type means evaluateMap() is called.